### PR TITLE
[Portsorch]: Add lag-name and oid map to counter table COUNTERS_PORT_NAME_MAP

### PR DESCRIPTION
**Pull request by leoli-nps**
**What I did**
Add the lag name and oid mapping to the counter table after the lag is created.
**Why I did it**
Can not obtain the name of lag by oid when using port_util.get_interface_oid_map(). Causes abnormal display when using show mac command after PortChannel0001 learns mac.
**How I verified it**
Use redis-cli as below:
127.0.0.1:6379[2]> HGET COUNTERS_PORT_NAME_MAP PortChannel0001
"oid:0x200000000060f"
127.0.0.1:6379[2]>
**Details if related**

### CODE REVIEW

File: orchagent/portsorch.cpp [hunk 1]
```diff
@@ -2232,6 +2232,12 @@ bool PortsOrch::addLag(string lag_alias)
     lag.m_members = set<string>();
     m_portList[lag_alias] = lag;
 
+    /* Add lag name map to counter table */
+    FieldValueTuple tuple(lag_alias, sai_serialize_object_id(lag_id));
+    vector<FieldValueTuple> fields;
+    fields.push_back(tuple);
+    m_counterTable->set("", fields);
```
Comment (lguohan) [inline orchagent/portsorch.cpp]: we need the remove part as well.

**Comment by leoli-nps** [comment]
The remove part is added, please  review!

File: orchagent/portsorch.cpp (hunk 1 above)
Comment (leoli-nps) [reply orchagent/portsorch.cpp]: The modification has been uploaded, please take a moment to review it. I want to continue uploading code here(sonic-swss), thanks.

**Comment by leoli-nps** [comment]
Hi stcheng, please take a moment to review this pull request, we need this pfc-wd function; look forward to your reply, be grateful.
@stcheng

**Comment by shine4chen** [comment]
@lguohan @pavel-shirshov

**Comment by lguohan** [comment]
the pfc wd code is not relavant is the PR. can you create a separate PR for the pfcwd support and for this PR only limit change to COUNTERS_PORT_NAME_MAP.

**Comment by leoli-nps** [comment]
Hi lguohan, I removed the pfc-wd code based on your suggestion. Now there is only COUNTERS_PORT_NAME_MAP related operation in this PR, please take a moment to deal with it, thanks.
@lguohan

**Comment by leoli-nps** [comment]
@lguohan Please help review this PR, thanks.

**Comment by leoli-nps** [comment]
Same as [PR#808](https://github.com/Azure/sonic-swss/pull/808) and [PR#885](https://github.com/Azure/sonic-swss/pull/885)

This PR can be closed based on that changes.

**Pull request closed without merging**